### ISSUE TITLE
Test frontend "looks"

### ISSUE THREAD

**Issue by niklasmohrin** [e-valuation/EvaP#2046]
If we want to pay more attention to how EvaP looks on mobile, we should come up a way to automatically check that our cool CSS keeps working in the future.

One possible approach would be screenshot tests, that is, automatically taking screenshots of rendered pages and comparing them against a previous screenshot (that is in the repository). Every PR that changes the CSS would probably also include an update of the screenshot (which reviewers could compare and see that it still looks okay). The downside is of course that PRs get bloated with updates to the screenshots and that they take up space in the git repo forever, which could lead to slow clones. We should investigate how other projects do this, one other project someone presented me had a separate branch for these screenshots and they were reviewed after the fact, but this doesn't seem optimal to me.

**Comment by niklasmohrin** [comment]
Good idea, it would be cool if the "do the screenshot" thing would be a script that I could also run locally to use with git bisect for example. 

Another thought: we could probably cache the screenshot of each commit for efficiency and for being able to inspect if from the GitHub actions panel. If this works like I am currently thinking it does, it seems like a nice workflow to me :D

**Comment by karyon** [comment]
Regarding the repo size concern, there is git-lfs, which generally just works, except that github [apparently ](https://github.com/orgs/community/discussions/15267)doesn't show diffs for images in git lfs. Which is quite a letdown.

**Comment by hansegucker** [comment]
With the current progress of implementing live tests with Selenium, we are able to programmatically create screenshots. Together with a tool like Argos CI or Percy, visual comparisons should be possible.
